packages/jdk.rb

```diff
@@ -1,43 +1,51 @@
 require 'package'
 
 class Jdk < Package
-  description 'The JDK is a development environment for building applications, applets, and components using the Java programming language.'
-  homepage 'https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html'
-  version '8u333-18.0.1.1'
+  description 'The Oracle JDK is a development environment for building applications, applets, and components using the Java programming language.'
+  homepage 'https://www.oracle.com/java'
   license 'Oracle-BCLA-JavaSE'
   compatibility 'all'
 
+  # full version number extracted from jdk*.rb recipes
+  @avail_jdk_ver = [8, 11, 17, 18].to_h do |jdk_majver|
```

Comment:
How about instead of an array of versions (still hardcoded), why not do something like this:
```
Dir["#{CREW_PACKAGES_PATH}/jdk*.rb"].each do |package|
  pkg = Package.load_package("#{CREW_PACKAGES_PATH}/#{package}")
  ...
```

Reply:
Change made